app/src/main/java/com/example/photogallery/MainActivity.java

```diff
@@ -34,7 +36,30 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_main);
         String currentPhotoPath;
 
+        //get the list of images paths
+        ArrayList<String> imagePaths = new ArrayList<>();
```

Comment:
Please declare `imagePaths` as a private variable, so we can reuse it in the search function.

Reply:
Can someone do it in Search function branch, I currently don't have an access to my dev environment.

Reply:
Nevermind, made it through bitbucket. Sorry for a late reply.